### PR TITLE
task 4 endpoint and tests

### PR DESCRIPTION
**Pull request by BTMurph0**

### CODE REVIEW

File: models/articles-model.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.selectArticle = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then((result) => {
+      if (!result.rows[0]) {
+        return Promise.reject({
```
Comment (nc-hannah-burks) [inline models/articles-model.js:8]: Nice error handling! 👏🏻

File: __tests__/app.test.js [hunk 2]
```diff
@@ -33,14 +33,48 @@ describe("GET /api/topics", () => {
...
+        expect(endPoints).toEqual(endpointsFile);
+      });
+  });
+});
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/7")
+      .expect(200)
+      .then((response) => {
```
Comment (nc-hannah-burks) [inline __tests__/app.test.js:52]: We also need to be checking the article id to check we have the right article back (7)

**Review by nc-hannah-burks — APPROVED** [review]
Great job Barry! Keep it up!

File: models/articles-model.js (hunk 1 above)
Comment (BTMurph0) [reply models/articles-model.js:8]: Thanks Hannah

File: __tests__/app.test.js (hunk 2 above)
Comment (BTMurph0) [reply __tests__/app.test.js:52]: oh yeah, of course

**Pull request merged**